### PR TITLE
Add AboutUs Page

### PR DESCRIPTION
**Pull request by neeta20**
New page created - "About"

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -58,6 +59,7 @@ const App = () => {
           <AppRoute exact path="/roadmap" component={RoadMap} layout={LayoutDefault} />
           <AppRoute exact path="/airdrop" component={AirDrop} layout={LayoutDefaultSlim} />
           <AppRoute exact path="/paper-gardens" component={Chapters} layout={LayoutDefaultSlim} />
+          <AppRoute exact path="/About" component={About} layout={LayoutDefault} />
```
Comment (way2rach) [inline src/App.js]: could we change the path to all lowercase?

File: src/components/layout/partials/FooterNav.js [hunk 2]
```diff
@@ -54,6 +54,14 @@ const FooterNav = ({
             Blog
           </ReactGA.OutboundLink>
         </li>
+
+        <li style={{marginBottom: "20px"}}>
+          <ReactGA.OutboundLink eventLabel="Footer About Link" to="/About">
```
Comment (way2rach) [inline src/components/layout/partials/FooterNav.js]: and this will then be to="/about"

File: src/components/sections/About.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    >
+      <div className="container">
+        <div className={innerClasses}>
+          <div className="hero-content">
+            <h1 className="mt-0 mb-16 reveal-from-bottom" data-reveal-delay="200" >
+            Our Story
+            </h1>
+           
+            <div className="container-sm" style={{marginTop: "50px", fontWeight: "500", color:"#9CA9B3", lineHeight: "30px", fontSize: "20px", textAlign: "left"}}>
+              <p className="m-0 mb-32 reveal-from-bottom" data-reveal-delay="400">John, one of the founders of Shrub had always loved the financial markets and being able to research companies and invest in them. For years he kept up this hobby, keeping up with all of the financial news and then buying an ownership stake in companies he believed were undervalued.</p>
```
Comment (way2rach) [inline src/components/sections/About.js]: and may be we remove all the data delays

File: src/App.js (hunk 1 above)
Comment (neeta20) [reply src/App.js]: path changed to lowercase

File: src/components/layout/partials/FooterNav.js (hunk 2 above)
Comment (neeta20) [reply src/components/layout/partials/FooterNav.js]: changed /About.  to /about

File: src/components/sections/About.js (hunk 3 above)
Comment (neeta20) [reply src/components/sections/About.js]: removed data delays by deleting data-reveal-delay="400"  from /about.js

Comment (way2rach) [reply src/components/sections/About.js]: may need to clean this guy up as well, and then we are ready to go 🚀 `reveal-from-bottom`

Comment (neeta20) [reply src/components/sections/About.js]: reveal-from-bottom removed

**Pull request merged**